starknet_py/tests/e2e/fixtures/accounts.py

```diff
@@ -0,0 +1,220 @@
+# pylint: disable=redefined-outer-name
+
+import sys
+from dataclasses import dataclass
+from typing import List, Tuple
+
+import pytest
+import pytest_asyncio
+
+from starknet_py.contract import Contract
+from starknet_py.hash.address import compute_address
+from starknet_py.net import KeyPair
+from starknet_py.net.account.account import Account
+from starknet_py.net.account.base_account import BaseAccount
+from starknet_py.net.full_node_client import FullNodeClient
+from starknet_py.net.gateway_client import GatewayClient
+from starknet_py.net.http_client import GatewayHttpClient
+from starknet_py.net.models import StarknetChainId
+from starknet_py.tests.e2e.fixtures.constants import (
+    DEVNET_PRE_DEPLOYED_ACCOUNT_ADDRESS,
+    DEVNET_PRE_DEPLOYED_ACCOUNT_PRIVATE_KEY,
+    INTEGRATION_NEW_ACCOUNT_ADDRESS,
+    INTEGRATION_NEW_ACCOUNT_PRIVATE_KEY,
+    TESTNET_NEW_ACCOUNT_ADDRESS,
+    TESTNET_NEW_ACCOUNT_PRIVATE_KEY,
```

Comment:
Should we rename now all of the `new_` variables? And make sure all of the `old_` ones are removed too.